src/arrayMethodSort.js

```diff
@@ -4,8 +4,33 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    let callBack = compareFunction;
+
+    if (!callBack) {
+      callBack = function (a, b) {
+        if (String(a) > String(b)) {
```

Comment:
It's not necessary to convert `a` and `b` to a string. JavaScript will automatically convert them to a string when comparing, so you can remove the `String()` part.